public/locales/en.json

```diff
@@ -37,6 +37,26 @@
   "connectWallet": "Connect Wallet",
   "copyAddress": "Copy",
   "copied": "Copied!",
+  "createPool": "Create Pool",
+  "createPoolDescription": "Create a community pool to set up token pairs of your choice for any pegged asset (up to 4 tokens). Community pools are user-created pools that will live on Saddle's Pools page under &ldquo;community pools,&ldquo; and accessible via API. Once the pool is created, you'll have to seed the pool with liquidity.",
+  "addPoolName": "Add pool name and symbol",
```

Comment:
@penandlim how do we assure uniqueness here? I guess we can do frontend validation, and the pool registry will revert if not unique?